src/Response/XmlApiResponse.php

```diff
@@ -18,18 +19,25 @@ class XmlApiResponse implements ApplicationIResponse
      */
     private $response = null;
 
+    /**
+     * @var string|int|bool|DateTimeInterface
```

Comment:
isn't this so wide? why not keep one of them and be more strict? I know that all of those can be used in `IResponse->setExpiration` but I personally don't like it to have too open

Reply:
Changed to DateTimeInterface|null, could we merge?